windows/installer-source/KaliteSetupScript.iss

```diff
@@ -29,7 +29,6 @@ SolidCompression=yes
 PrivilegesRequired=admin
 UsePreviousAppDir=yes
 ChangesEnvironment=yes
-AlwaysRestart=yes

```

Comment:
No, turns out we can just disable the option to start the program directly from the installer.